govcd/common_test.go

```diff
@@ -230,10 +230,10 @@ func testCheckLoadBalancerConfig(beforeLb *types.LbGeneralParamsWithXml, beforeL
 // reached
 func isTcpPortOpen(host, port string, timeout int) bool {
 	retryTimeout := timeout
-	// due to the VMs taking long time to boot it needs to be at least 5 minutes
+	// due to the VMs taking long time to boot it needs to be at least 6 minutes
 	// may be even more in slower environments
-	if timeout < 5*60 { // 5 minutes
-		retryTimeout = 5 * 60 // 5 minutes
+	if timeout < 6*60 { // 5 minutes
```

Comment:
```suggestion
	if timeout < 6*60 { // 6 minutes
```
or we can get rid of these comments as the same info is on line 233 as well.

Reply:
Removed it in fact. Even the syntax is quite clear `6 * 60` + there is one explanation comment.